en-US/Grammar.xml

```diff
@@ -146,6 +146,76 @@
 		</table>
 
 	</section>
+	<section id="possessives">
+	<title>Possessives</title>
+	<para>
+	A <emphasis>possessive</emphasis> indicates that something or someone belongs to a person or thing.
+	<!-- To form a possessive generally, for a singular noun add an apostrophe and <emphasis>s</emphasis>; for a plural noun, add an apostrophe after the final letter <emphasis>s</emphasis>. -->
+	</para>
+	 <para>
+	For guidance on how to form a possessive, see <xref linkend="apostrophes"/>.
+	</para>
+	<para>
+	Consider the following scenarios for when to use or not to use a possessive:
+	</para>
+	<para> 
+	Do not use possessives to refer to abbreviations or product names.
```

Comment:
Since abbreviations are mentioned here, I suggest updating the relevant section, 3.6. Using Abbreviations, Acronyms, Initialisms, and Special Characters Correctly, with this guidance as well. That section is currently being updated, but I suggest adding the lines: "Do not use possessives with abbreviations. For examples and more information, see <xref linkend="possessives"/>." I'm tagging @julian-cable because he has worked on that section.

Point of discussion: should this guidance be expanded to include initialisms? If yes, we could say: "Do not use possessives with abbreviations and initialisms" followed by the link to the new possessives section.

Reply:
We should also add a third piece of guidance with examples: "Do not use possessives with company names." followed by examples. Putting my previous suggestions together, I suggest ordering the guidance in this section (with their examples) based on its relevance to Red Hat training materials: company name, product names, abbreviations.